### ISSUE TITLE
Verification sms not received.

### ISSUE THREAD

**Issue by ken-de-moor** [threefoldtech/threefold_connect#263]
Succesfully validated the same number on 18 June, today 28th June it's failing to validate.

Not really any logging to show as the app doesn't error or anything, SMS just isn't received.

**Comment by ken-de-moor** [comment]
Twilio service was drained, topped up again now.

**Comment by sabrinasadik** [comment]
Issue resolved for all of the users except one: 1-818-903-8380
still not receiving SMS verification code. User from California.

**Comment by sabrinasadik** [comment]
More users from US not receiving the phone verification

**Comment by sabrinasadik** [comment]
We need updates on this, asap.

**Comment by FlorinaRogers** [comment]
@JonasWijne @sabrinasadik  

Please find below the list of user's with the not receiving phone number verification sms issue, their locations, and their phone numbers as gathered from Crisp chat:

1) Username: Michael 
    Email: 4msc22@gmail.com
    Location: Calabasas, CA
    Phone number: 1-818-903-8380

2) Username: Tuckerbrooks79 
    Email: tuckerbrooks79@gmail.com
    Location: Richardson, TX
    Phone number: 1-505-320-6186

3) Username: Chamcham41 
    Email: chamcham41@yahoo.com
    Location: Orlando, FL
    Phone number: 1-352-362-9018

4) Username: Damien Bertrand 
    Email: damien.bertrand@powerontechnology.com
    Location: Los Angeles, CA
    Phone number: Waiting for the user to provide.

5) Username: Sjacobs525 
    Email: sjacobs525@gmail.com
    Location: Las Vegas, NV
    Phone number: Waiting for the user to provide.

6) Username: Mmursch2015 
    Email: mmursch2015@gmail.com
    Location: Cape Coral, FL
    Phone number: 1-239-286-3836

7) Username: James 
    Email: jamesfinan@outlook.com
    Location: Schiedam
    Phone number: Waiting for the user to provide.

8) Username: Jon 
    Email: jon@hatchery.no
    Location: Kongsberg
    Phone number: Waiting for the user to provide.
    User Comments: Phone confirmation for Norway don't work. it asks for a number one digit longer than the numbers in     Norway...

**Comment by jdelrue** [comment]
Thanks, will followup with a user!

**Comment by JonasWijne** [comment]
i din't see them in twillio, not even an error, but when i checked the openkyc logs i get the following error 
![image](https://user-images.githubusercontent.com/6184820/126145097-7a2c70c1-e993-4e5a-b782-7d097ec9c08f.png)

**Comment by sasha-astiadi** [comment]
@scottyeager could you please vrify with US number?

**Comment by scottyeager** [comment]
I tried with my US number and did not receive an SMS.

This issue is public... shouldn't we be logging users' private data in a private place?

**Comment by sabrinasadik** [comment]
Support repo is private, but this one seems to be public. @FlorinaRogers can you please create a new issue in the support repo where we have all relevant private details? We can link that issue with this one afterward.

**Comment by sabrinasadik** [comment]
See details and phone numbers: https://github.com/threefoldtech/tf_support/issues/14#event-5043308662

**Comment by FlorinaRogers** [comment]
> 
> 
> Support repo is private, but this one seems to be public. @FlorinaRogers can you please create a new issue in the support repo where we have all relevant private details? Delete that information from this issue then. We can link that issue with this one afterward.

Sabrina, the issue has been created as requested in Support repo and assigned to you.

**Comment by scottyeager** [comment]
Thank you @sabrinasadik and @FlorinaRogers :pray:

**Comment by FlorinaRogers** [comment]
> 
> 
> Thank you @sabrinasadik and @FlorinaRogers 🙏

You are very welcome Scotty :)

**Comment by sabrinasadik** [comment]
Any status update?

**Comment by FlorinaRogers** [comment]
@JonasWijne and @ken-de-moor 


Please give an update on this issue.

**Comment by jdelrue** [comment]
@JonasWijne added a US number, so should normally work now. Will check with US residence ASAP.

**Comment by jdelrue** [comment]
Fixed and tested with @scottyeager. Issue can be closed and users notified, should work now.

**Comment by jdelrue** [comment]
can we close @FlorinaRogers ?

**Comment by FlorinaRogers** [comment]
I am still waiting on a few users to write back @jdelrue

**Comment by FlorinaRogers** [comment]
@jdelrue You can close this issue.
